flight/Modules/Stabilization/stabilization.c

```diff
@@ -320,11 +367,18 @@ static void stabilizationTask(void* parameters)
 		bool error = frequency_wrong;
 
 		if (gyro_filter_updated) {
-			if (settings.GyroCutoff < 1.0f) {
-				gyro_alpha = 0;
-			} else {
-				gyro_alpha = expf(-2.0f * (float)(M_PI) *
-						settings.GyroCutoff * dT_expected);
+			if(!butter_gyro_filter)
+			{
+				if (settings.GyroCutoff < 1.0f) {
+					gyro_alpha = 0;
+				} else {
+					gyro_alpha = expf(-2.0f * (float)(M_PI) *
+							settings.GyroCutoff * dT_expected);
+				}
+			}
+			else
```

Comment:
Our coding convetion places brackets on the same line as if / else / everything else (other than function decls).